config.js

```diff
@@ -10,12 +10,26 @@ window.REACT_APP_GRAPHQL_ENABLE_SSL = false;
 
 
 // Google API Key
-window.REACT_APP_GOOGLE_MAPS_KEY = "AIzaSyBM_PVN-MSaiayT2alT2dikHbmZbXeBy4s";
+window.REACT_APP_GOOGLE_MAPS_KEY = "AIzaSyBl8J85KibSuhHEw_j0JTlITVTBb8oXiMo";
 
-//Authentications
-window.REACT_APP_AUTH_SERVER = "https://auth.mint.isi.edu/";
-window.REACT_APP_AUTH_REALM = "production";
-window.REACT_APP_AUTH_CLIENT_ID = "mint-ui";
+// TAPIS
+window.REACT_APP_AUTH_PROVIDER = "tapis";
```

Comment:
document the available options. If one of the option is keycloak, please rename to oauth2 or generic.